static/app/components/modals/addDashboardWidgetModal.tsx

```diff
@@ -134,9 +135,9 @@ const newIssueQuery: WidgetQuery = {
 
 const newMetricsQuery: WidgetQuery = {
   name: '',
-  fields: [`sum(${SessionMetric.SESSION})`],
+  fields: [`sum(session)`],
```

Comment:
there are still a couple of places where we can replace this enum https://github.com/getsentry/sentry/blob/a838ad4f4fe98b0e9a70144792af5bed099626b2/static/app/utils/metrics/fields.tsx#L5